### ISSUE TITLE
Not working ?

### ISSUE THREAD

**Issue by echivalent** [JoseTomasTocino/image-metadata-viewer#4]
In the past days, I noticed that the app is not working. Happened before but after a while, everthing was fine. Is all good?

**Comment by JoseTomasTocino** [comment]
No, it’s not. Heroku was the hosting service I was using to host the viewer and it’s no longer free. Therefore it will remain offline until I find a similar free hosting service. I’m open to suggestions.

**Comment by echivalent** [comment]
Ok, i think i can provide hosting, how we proceed?
